main.go

```diff
@@ -112,7 +114,12 @@ var (
 	seed []byte
 )
 
-const garbleMapFile = "garble.map"
+const (
+	garbleMapFile      = "garble.map"
+	minGoVersion       = 1.15
+	maxGoVersion       = 1.16
+	supportedGoVersion = "1.15.x"
```

Comment:
rename to supportedGoVersions

Because 1.15.x are multiple versions,
also in the future we may have a range like
```
supportedGoVersions = "1.15.x - 1.17.x"
```